testsuite/setup.py

```diff
@@ -185,6 +185,7 @@ def run(self):
               'hypothesis',
               'psutil>=4.0.2',
               'mock>=2.0.0',  # replace with unittest.mock in python 3 only version
+              'tidynamics>=1.0.0'
```

Comment:
I am not 100% sure if we need the package for the test setup.py if it is already required by MDA itself. — @richardjgowers @tylerjereddy anybody else ?

Reply:
Given that it's an analysis dependency and one can install MDAnalysis without it but the tests require it, it has to stay here.

Please also raise an issue for the conda-forge MDAnalysisTests package so that it is made a required dependency there. Otherwise this whole test module will fail with ImportError.